conanfile.py

```diff
@@ -131,6 +133,7 @@ def generate(self):
         tc.cache_variables["CLOE_ENGINE_WITH_SERVER"] = self.options.engine_server
         tc.cache_variables["CLOE_ENGINE_WITH_LRDB"] = self.options.engine_lrdb
         tc.cache_variables["CLOE_PYTHON_API"] = self.options.python_api
+        tc.cache_variables["PYTHON_BINDINGS_LOCAL_DEV"] = not self.in_local_cache
```

Comment:
In general, we try to write the CMakeLists.txt in such a way that it doesn't know about Conan. Here I would just use a `PYTHING_BINDING_OUTPUT_DIRECTORY` which specifies the path, and then later use the path if its defined.

Reply:
I'm going to use `PYTHONPATH` to locate the `_cloe_bindings` so file.